### ISSUE TITLE
Testing: Update e2e tests to include a11y helper

### ISSUE THREAD

**Issue by jcfranco** [Esri/calcite-app-components#385]
With #37 installed, e2e tests for all components should be updated to cover a11y.

@driskull 

- [x] `calcite-action` 
- [x] `calcite-action-bar`
- [x] `calcite-action-group`
- [x] `calcite-action-pad`
- [x] `calcite-flow`
- [x] `calcite-flow-item`
- [x] `calcite-panel`
- [x] `calcite-shell`
- [x] `calcite-shell-panel`

@jcfranco 

- [x] `calcite-block`
- [ ] `calcite-block-section`
- [x] `calcite-example`

@pr3tori4n 

- [x] `calcite-filter`
- [x] `calcite-pick-list`
- [x] `calcite-pick-list-group`
- [x] `calcite-pick-list-item`
- [x] `calcite-tip`
- [x] `calcite-tip-group`
- [x] `calcite-tip-manager`
- [x] `calcite-value-list`
~~- [ ] `calcite-value-list-item`~~ doesn't have any tests since it's basically just a wrapping container around `pick-list-item`.

**Comment by pr3tori4n** [comment]
@jcfranco @driskull 
Can we either put component assignments in the first comment for each component or split this into 3 issues, assigned to us individually with the components we each should handle?

**Comment by jcfranco** [comment]
> Can we either put component assignments in the first comment for each component 

👍Please review and update assignments if needed.

**Comment by pr3tori4n** [comment]
`value-list-item` doesn't have any tests since it's basically just a wrapping container around `pick-list-item`.

**Comment by pr3tori4n** [comment]
@jcfranco please verify.

**Comment by jcfranco** [comment]
I missed setting up a test for `calcite-block-section`. I'll create a separate issue for this. Aside from that, all components are using the `accessible` helper.
